version.php

```diff
@@ -27,7 +27,7 @@
 
 defined('MOODLE_INTERNAL') || die;
 
-$plugin->version   = 2024112001;      // The current plugin version (Date: YYYYMMDDXX).
+$plugin->version   = 2025030600;      // The current plugin version (Date: YYYYMMDDXX).
```

Comment:
this should be a micro bump so just 2024112002

Reply:
fixed